### PR TITLE
:art:Added a conversational hero section

### PR DESCRIPTION
**Pull request by teddy5456**
# Description

After creation of the conversational-learning page, i added a hero section just below the header. I created a separate styling file with css under `/src/features/styles/conversational-learning-page`

fixes issue #79 
Dependent on issue #10 #1

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Screenshot (optional)
image
### smaller screens (798px)
image

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/features/pages/conversational-learning-page.html [hunk 1]
```diff
@@ -0,0 +1,462 @@
...
+h3 {
+    margin-top: 0;
+    left: 209px;
+    width: 701px;
+    height: 224px;
+    text-align: left;
+    font: normal normal normal 32px/42px DM Sans;
+    letter-spacing: 0px;
+    color: #000000;
+    opacity: 1;
```
Comment (JAPHETHNYARANGA) [inline src/features/pages/conversational-learning-page.html]: here and rest of the style

**Review by JAPHETHNYARANGA — CHANGES_REQUESTED** [review]
This is very good work, just a small change then i merge. create the separate style sheet then move your styles there , then ask me to merge. If you are around feel free to walk up to me

File: src/features/pages/conversational-learning-page.html (hunk 1 above)
Comment (teddy5456) [reply src/features/pages/conversational-learning-page.html]: @JAPHETHNYARANGA  i have made the changes requested. I request a review please.

**Comment by JAPHETHNYARANGA** [comment]
Good attempt closing this pr as it is already done, please pick another issue

**Pull request closed without merging**